examples/relying_party/dumps/example.json

```diff
@@ -63,6 +63,9 @@
     "trust_marks_issuers": {},
     "entity_type": "openid_relying_party",
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8001/resolve"
```

Comment:
can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts